apps/api/src/deployment/repositories/deployment/deployment.repository.ts

```diff
@@ -0,0 +1,46 @@
+import { Deployment, Lease } from "@akashnetwork/database/dbSchemas/akash";
+import { literal, Op } from "sequelize";
+import { singleton } from "tsyringe";
+
+export interface StaleDeploymentsOptions {
+  createdHeight: number;
+  owner: string;
+}
+
+export interface StaleDeploymentsOutput {
+  dseq: number;
+}
+
+@singleton()
+export class DeploymentRepository {
+  async findStaleDeployments(options: StaleDeploymentsOptions): Promise<StaleDeploymentsOutput[]> {
+    const deployments = await Deployment.findAll({
+      attributes: ["dseq"],
+      include: [
+        {
+          model: Lease,
+          attributes: [],
+          required: false
+        }
+      ],
+      where: {
+        owner: options.owner,
+        [Op.and]: [
+          {
+            createdHeight: {
+              [Op.lt]: options.createdHeight
+            }
+          },
+          {
+            closedHeight: null
+          }
+        ]
```

Comment:
Could the conditions be all on the same level like this?
```
{
        owner: options.owner,
        createdHeight: {
          [Op.lt]: options.createdHeight
        },
        closedHeight: null
}
```

Reply:
Yea, you're right, will fix with another pr. merged this before I saw this comment